Makefile

```diff
@@ -0,0 +1,56 @@
+# **************************************************************************** #
+#                                                                              #
+#                                                         :::      ::::::::    #
+#    Makefile                                           :+:      :+:    :+:    #
+#                                                     +:+ +:+         +:+      #
+#    By: zslowian <zslowian@student.42.fr>          +#+  +:+       +#+         #
+#                                                 +#+#+#+#+#+   +#+            #
+#    Created: 2024/12/04 15:20:00 by zslowian          #+#    #+#              #
+#    Updated: 2024/12/16 19:18:26 by zslowian         ###   ########.fr        #
+#                                                                              #
+# **************************************************************************** #
+
+NAME = pipex
```

Comment:
Please adjust to represent minishell name instead of pipex (for the whole project)

Reply:
"pipex" naming left only in README - created a separate task to clean-up readme (as it is not crucial at this moment)